bedrock/firefox/templates/firefox/facebookcontainer/index.html

```diff
@@ -33,40 +33,58 @@
     </div>
 
     <div class="mobile-cta">
-      <p>{{ ftl('facebook-container-the-facebook-container-extension') }}</p>
-      <p>{{ ftl('facebook-container-try-firefox-focus-the-privacy') }}</p>
-
-      <ul>
-        <li>
-          {{ google_play_button(href=focus_adjust_url('android', 'fb-container-page'), id='playStoreLink', product='Focus') }}
-        </li>
-        <li>
-          <a id="appStoreLink" href="{{ focus_adjust_url('ios', 'fb-container-page') }}" data-link-type="download" data-download-os="iOS">
-            <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ ftl('download-button-download-app-store') }}" width="152" height="45">
-          </a>
-        </li>
-      </ul>
+      {% if ftl_has_messages('facebook-container-only-available-for-desktop', 'facebook-container-visit-to-get-for-desktop') %}
+        <p>{{ ftl('facebook-container-only-available-for-desktop') }}</p>
+        <p>{{ ftl('facebook-container-visit-to-get-for-desktop', url=url('firefox.new'), link_copy='www.mozilla.org/firefox/new/') }}</p>
+      {% else %}
+        <p>{{ ftl('facebook-container-the-facebook-container-extension') }}</p>
+      {% endif %}
+
+      {% if ftl_has_messages('facebook-container-get-firefox-android-ios') %}
+        <p>{{ ftl('facebook-container-get-firefox-android-ios') }}</p>
+
+        <ul>
+          <li>
+            {{ google_play_button(href=firefox_adjust_url('android', 'fb-container-page'), id='playStoreLink') }}
+          </li>
+          <li>
+            <a id="appStoreLink" href="{{ firefox_adjust_url('ios', 'fb-container-page') }}" data-link-type="download" data-download-os="iOS">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ ftl('download-button-download-app-store') }}" width="152" height="45">
+            </a>
+          </li>
+        </ul>
+      {% else %}
```

Comment:
Maybe there shouldn't be a fallback here? I think it would be okay to end with "The Facebook Container Extension is not available on mobile devices" without providing the Focus CTA after it.

Reply:
Good call - I'll remove the else 👍